lightning_pose_app/ui/extract_frames.py

```diff
@@ -490,3 +525,8 @@ def _render_streamlit_fn(state: AppState):
 
         # force rerun to show "waiting for existing..." message
         st_autorefresh(interval=2000, key="refresh_extract_frames_after_submit")
+
+    st.divider()
+
+    st.header("Placeholder for frames upload element")
+    
```

Comment:
can remove this for now since we're dealing with this on the other branch

Reply:
Amended